symengine/polys/basic_conversions.h

```diff
@@ -0,0 +1,221 @@
+#include <symengine/visitor.h>
+
+namespace SymEngine
+{
+
+template <typename P>
+RCP<const P> from_basic(const RCP<const Basic> &basic,
+                        const RCP<const Basic> &gen, bool ex = false);
+template <typename P>
+RCP<const P> from_basic(const RCP<const Basic> &basic, bool ex = false);
```

Comment:
Can you add some documentation about these two methods? (A comment above each method would do)